lib/mjml.rb

```diff
@@ -29,6 +31,15 @@ module Mjml
                                   ' have you run $ npm install mjml?'
   self.beautify = true
   self.minify = false
+  self.fonts = {
```

Comment:
Rubocop made me write default font hash in multiline, but as a consequence, Module is now too long (this explains `Metrics/ModuleLength` disabling)

Reply:
ok, I thought it was a requirement to mimic the configuration used to setup the other options ! I'll be happy to lighten it